core-services/prow/02_config/_config.yaml

```diff
@@ -3076,6 +3076,28 @@ tide:
     - do-not-merge/invalid-owners-file
     repos:
     - openshift/release
+  - excludedBranches:
+    - release-3.11
+    - release-4.0
+    - release-4.1
+    - release-4.2
+    - release-4.3
+    - release-4.4
+    - release-4.5
+    - release-4.6
+    - release-4.7
```

Comment:
I'm not 100% sure this was the most elegant way to tackle this, but essentially we don't have a concrete plan yet for what our branches will be named in these upstream repositories yet as we move off the OCP release cadence. This seemed like the most straightforward way to get this working.